java/pom.xml

```diff
@@ -763,6 +766,60 @@
       </build>
     </profile>
 
+    <profile>
+      <id>error-prone-jdk11+</id>
+      <activation>
+        <jdk>[11,)</jdk>
+        <property>
+          <name>!m2e.version</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <source>8</source>
+              <target>8</target>
+              <encoding>UTF-8</encoding>
+              <compilerArgs combine.children="append">
+                <arg>-XDcompilePolicy=simple</arg>
+                <arg>
+                  -Xplugin:ErrorProne \
+                  -XepExcludedPaths:.*/(target/generated-sources)/.*
```

Comment:
nit, but can these be two separate `<arg>` blocks or does it not work that way?

Reply:
It does not work on that way.

Be aware that when running on JDK 8 the flags cannot be wrapped across multiple lines. JDK 9 and above do allow the flags to be separated by newlines. That is, [the second <arg> element above can also be formatted as follows](https://errorprone.info/docs/flags) on JDK 9+, but not on JDK 8:

Reply:
Ah, weird.